changelogs/8.15.asciidoc

```diff
@@ -1,9 +1,21 @@
 [[apm-release-notes-8.15]]
 == APM version 8.15
 
+* <<apm-release-notes-8.15.2>>
 * <<apm-release-notes-8.15.1>>
 * <<apm-release-notes-8.15.0>>
 
+[float]
+[[apm-release-notes-8.15.2]]
+=== APM version 8.15.2
```

Comment:
seems to be missing bugfix https://github.com/elastic/apm-data/pull/346 and feature https://github.com/elastic/apm-data/pull/339

Reply:
Thanks I was not aware of these, didn't spot them in the compare

Reply:
Updated!